### ISSUE TITLE
On Noise Sensitivity and Learning Rate

### ISSUE THREAD

**Issue by edward3862** [River-Zhang/ICEdit#83]
Hello authors, thanks for your amazing work!

I noticed that when using ICEdit for editing, the output can sometimes be highly sensitive to noise. Is this expected behavior?

Also, I saw in your provided configuration files that the learning rate for fine-tuning is set to 1. Have you experimented with different learning rates, and does such a high value lead to better performance?

Thanks:)

**Comment by HorizonWind2004** [comment]
One of our contribution is an inference-time scaling strategy~
We use Prodigy optimizer and its recommended lr is 1.
